src/dlgTriggerEditor.cpp

```diff
@@ -8044,3 +8050,39 @@ void dlgTriggerEditor::slot_clearSearchResults()
     textRanges->clear();
     controller->update();
 }
+
+// shows a custom right-click menu for the editor, including the indent action
+void dlgTriggerEditor::slot_editorContextMenu()
+{
+    // retrieve the current controller and editor
+    edbee::TextEditorWidget* editor = mpSourceEditorEdbee;
+    if (!editor) {
+        return;
+    }
+
+    edbee::TextEditorController* controller = mpSourceEditorEdbee->controller();
+
+    // create the menu
+    auto menu = new QMenu();
+    menu->addAction(controller->createAction("cut", tr("Cut"), QIcon(), menu));
```

Comment:
Do we need to do anything/want to bother with adding keyboard short-cuts?

Reply:
These already do get keyboard shortcuts!